src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logotype">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#samsung" class="nav-link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#smartphones" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#laptops & komputers"
+              data-qa="hover"
+              class="nav-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#gadgets" class="nav-link">Gadgets</a>
+          </li>
+          <li class="nav-item">
+            <a href="#tablets" class="nav-link">Tablets</a>
+          </li>
+          <li class="nav-item">
+            <a href="#photo" class="nav-link">Photo</a>
+          </li>
+          <li class="nav-item">
+            <a href="#video" class="nav-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
   </body>
-</html>
+</html>
```

Comment:
You should add an empty line at the end of each file 
```suggestion
</html>

```